ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.13-nightly-x86-loaded-upgrade-from-4.12.yaml

```diff
@@ -39,9 +39,11 @@ tests:
       OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE: release:latest
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
+      CONTROL_PLANE_INSTANCE_TYPE: m6i.xlarge
       ENABLE_FORCE: "true"
       GC: "false"
-      MAX_UNAVAILABLE: "3"
+      ITERATION_MULTIPLIER_ENV: "6"
+      MAX_UNAVAILABLE: "2"
```

Comment:
@liqcui Are these changes based on results analysis?
I am curious why we are introducing these 2 changes now.

Reply:
This should copy from other case, restore to 3 now!